CONTRIBUTING.md

```diff
@@ -41,7 +41,7 @@ That's it! 🎉 Others will be able to discover this provider much more easily n
 
 You can look at the existing built-in providers for inspiration.
 - If you are adding a new OIDC provider, take a look at [auth0.ts](https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers/auth0.ts).
-- If you are adding a new OAuth 2.0 provider, take a look at [google.ts](https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers/google.ts).
+- If you are adding a new OAuth 2.0 provider, take a look at [facebook.ts](https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers/facebook.ts).
```

Comment:
Changed to GitHub to match documentation.
https://authjs.dev/guides/configuring-oauth-providers#adhere-to-our-code-conventions

However, the GitHub does not use generics. I do not know the reason for this.
https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers/facebook.ts#L85C33-L85C60
https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers/github.ts#L124C25-L124C31
```suggestion
- If you are adding a new OAuth 2.0 provider, take a look at [github.ts](https://github.com/nextauthjs/next-auth/blob/main/packages/core/src/providers/github.ts).
```